Templates/AssetGem/template.json

```
@@ -1,38 +1,36 @@
 {
     "template_name": "AssetGem",
-    "origin": "The primary repo for AssetGem goes here: i.e. http://www.mydomain.com",
-    "license": "What license AssetGem uses goes here: i.e. https://opensource.org/licenses/MIT",
-    "display_name": "AssetGem",
-    "summary": "A short description of AssetGem template.",
-    "canonical_tags": [],
+    "template_restricted_platform_relative_path": "Templates/AssetGem",
+    "origin": "https://github.com/o3de/o3de",
+    "license": "https://opensource.org/licenses/Apache-2.0 Or https://opensource.org/licenses/MIT",
+    "display_name": "Asset Gem Template",
+    "summary": "Use this gem template to create the minimal gem needed for adding asset(s).",
+    "canonical_tags": [
+		"Template",
+        "Gem",
+        "Asset"
```

Nitpick: Spacing the "Template" line has a tab in it
```suggestion
        "Template",
        "Gem",
        "Asset"
```